ads/empower.md

```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Empower
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+	  type="empower"
+	  data-site="example"
+	  data-zone="123456">
+    <div placeholder></div>
+    <div fallback></div>
+</amp-ad>
+```
+
+
+## Configuration
+
+For details on the configuration semantics, please contact the ad network or refer to their documentation.
+
+### Required parameters
```

Comment:
Did you want to document `category` as an optional param here? You can also pass optional params as a third argument to `validateData()`.

Reply:
Exactly, didn't know about the third argument. Just added! 👍

Reply:
Sorry I wasn't clear, did you want to also add it to the doc under optional parameters. See example [here](https://github.com/ampproject/amphtml/blob/master/ads/_ping_.md#optional-parameters)

Reply:
Sorry, forgot to update the documentation.

Just added the **Optional Parameters** section with the `data-category` listed under it.